app/views/CannedResponsesListView/Dropdown/index.tsx

```diff
@@ -14,92 +13,85 @@ import { ILivechatDepartment } from '../../../definitions/ILivechatDepartment';
 const ANIMATION_DURATION = 200;
 
 interface IDropdownProps {
-	theme?: TSupportedThemes;
 	currentDepartment: ILivechatDepartment;
 	onClose: () => void;
 	onDepartmentSelected: (value: ILivechatDepartment) => void;
 	departments: ILivechatDepartment[];
 }
 
-class Dropdown extends React.Component<IDropdownProps> {
-	private animatedValue: Animated.Value;
+const Dropdown = ({ currentDepartment, onClose, onDepartmentSelected, departments }: IDropdownProps) => {
+	const animatedValue = useRef(new Animated.Value(0)).current;
+	const { theme } = useTheme();
 
-	constructor(props: IDropdownProps) {
-		super(props);
-		this.animatedValue = new Animated.Value(0);
-	}
-
-	componentDidMount() {
-		Animated.timing(this.animatedValue, {
+	useEffect(() => {
+		Animated.timing(animatedValue, {
 			toValue: 1,
 			duration: ANIMATION_DURATION,
 			easing: Easing.inOut(Easing.quad),
 			useNativeDriver: true
 		}).start();
-	}
+	}, [animatedValue]);
 
-	close = () => {
-		const { onClose } = this.props;
-		Animated.timing(this.animatedValue, {
+	const close = () => {
+		Animated.timing(animatedValue, {
 			toValue: 0,
 			duration: ANIMATION_DURATION,
 			easing: Easing.inOut(Easing.quad),
 			useNativeDriver: true
 		}).start(() => onClose());
 	};
 
-	render() {
-		const { theme, currentDepartment, onDepartmentSelected, departments } = this.props;
-		const heightDestination = 0;
-		const translateY = this.animatedValue.interpolate({
-			inputRange: [0, 1],
-			outputRange: [-300, heightDestination] // approximated height of the component when closed/open
-		});
-		const backdropOpacity = this.animatedValue.interpolate({
-			inputRange: [0, 1],
-			outputRange: [0, themes[theme!].backdropOpacity]
-		});
+	const heightDestination = 0;
+	const maxRows = 5;
```

Comment:
Constants should live outside the component, so they won't be recreated.

Reply:
@diegolmello Do I need to capitalise them?